.eslintrc.json

```
@@ -157,6 +157,37 @@
       "rules": {
         "filenames-simple/naming-convention": "off"
       }
+    },
+    {
+      "files": ["**/api/src/**/*.ts"],
+      "plugins": ["eslint-plugin-jsdoc"],
```

```suggestion
      "plugins": ["jsdoc"],
```
Does the same thing, but is shorter :man_shrugging: